data/load-seed-data.js

```diff
@@ -23,12 +23,11 @@ async function run() {
     );
       
     const user = users[0].rows[0];
-    console.log(todos);
 
     await Promise.all(
       todos.map(todo => {
         return client.query(`
-                    INSERT INTO todo (todo, completed, user_id)
+                    INSERT INTO todos (todo, completed, user_id)
```

Comment:
I know we went over this yesterday, but usually just use these as plural bc there will end up being multiple todo objects.  Good job fixing that!